modal/app.py

```diff
@@ -212,6 +213,10 @@ def description(self) -> Optional[str]:
         """The App's `name`, if available, or a fallback descriptive identifier."""
         return self._description
 
+    @property
+    def client(self) -> Optional[_Client]:
+        return self._client
```

Comment:
This will expose the client as public (i.e. user-facing) API, right? Are there any downsides to that?

Reply:
yeah good point. I did this for a not great reason—wanted to have `inferred_app` be a union of `App` and `RunningApp`, in a way that the fields we want are on both. And it didn't quite make sense to add `_client` on `RunningApp`. 

I'll just write it to explicitly if/else. Uglier but better.